spec/jira_api_spec.rb

```diff
@@ -0,0 +1,28 @@
+# -- coding: utf-8
```

Comment:
Could you send issue? https://github.com/embulk/embulk/blob/master/embulk-core/src/main/java/org/embulk/jruby/JRubyScriptingModule.java#L66

Should be 2.0!